database/sqlite/sqlite_functions.c

```diff
@@ -32,6 +32,9 @@ const char *database_config[] = {
     "repeat text, host_labels text, p_db_lookup_dimensions text, p_db_lookup_method text, p_db_lookup_options int, "
     "p_db_lookup_after int, p_db_lookup_before int, p_update_every int);",
 
+    "CREATE TABLE IF NOT EXISTS host_info(host_id blob, system_key text NOT NULL, system_value text NOT NULL, "
```

Comment:
I took a look in the arguments that are storing data in this table and I observed that the ideal scenario would be:

- A table that has all `system_info` and this must be the primary key.
- Another table to store all `host_id` that is also a primary key.
- Finally this table is adapted to use foreign keys from these tables, because we need a relationship `M X N` here.

Considering that we also have `host_id` in other tables, probably this will need another PR to reorganize all database and avoid unnecessary duplicated data.